pypeit/core/transform.py

```diff
@@ -0,0 +1,140 @@
+"""
+Provide basic coordinate tranformation functions.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+
+from IPython import embed
+
+import numpy as np
+
+from pypeit import msgs
+
+
+def affine_transform_matrix(scale=None, rotation=None, translation=None):
+    r"""
+    Construct an affine transformation matrix for a two-dimensional image in
+    homologous coordinates.
+
+    Method currently does *not* allow for a shear term.  The transformation is
+    returned as a `numpy.ndarray`_ in homologous coordinates.
+    
+    When applied to an image, this follows the convention that coordinates are
```

Comment:
Pease state explicitly in these docs and all similar docs which axis of a python array is x and which axis is y, since this is ambiguous for numpy arrays and often a big source of confusion. Namely, numpy arrays follow a matrix convention (first axis is row, second axis is column) which is the opposite of the Cartesian convention (x,y) that one would normally adopt for images.  I think you can just add as sentence stating more explicitly that (x,y) here refers to  an array of dimension shape = (nx,ny). You say first axis second axis, but that is not as clear as it could be. Keep also in mind that the ginga convention is flipped with respect to this.

Reply:
@kbwestfall just a reminder to address this comment please.